zest/releaser/pypi.py

```diff
@@ -274,12 +278,37 @@ class ZestReleaserConfig:
     hooks_filename = None
 
     def load_configs(self, pypirc_config_filename=DIST_CONFIG_FILE):
+        """Load configs from several files.
+
+        The order is this:
+
+        - ~/.pypirc
+        - setup.cfg
+        - pyproject.toml
+
+        A later config file overwrites keys from an earlier config file.
+        I think this order makes the most sense.
+        Example: extra-message = [ci skip]
+        What I expect, is:
+
+        * Most packages won't have this setting.
+        * If you make releases for lots of packages, you probably set this in
+          your global ~/.pypirc.
+        * A few individual packages will explicitly set this.
+          They will expect this to have the effect that the extra message is
+          added to commits, regardless of who makes a release.
+          So this individual package setting should win.
+        * Finally, pyproject.toml is newer than setup.cfg, so it makes sense
+          that this file has the last say.
+        """
         setup_config = SetupConfig()
         pypi_config = PypiConfig(config_filename=pypirc_config_filename)
         pyproject_config = PyprojectTomlConfig()
         combined_config = {}
-        # overwrite any duplicate keys in the following order:
-        for config in [setup_config, pypi_config, pyproject_config]:
+        config_files = [pypi_config]
+        if self.use_package_config:
+            config_files.extend([setup_config, pyproject_config])
```

Comment:
I'm wondering what this `use_package_config` setting does. It might need explaining in the `__init__()`.

..... Reinout reads the tests .....

Aha! this is only for during the test. `omit_package_config_in_test` would be clearer (default `False`, I think the negative term is clearer here).

Reply:
Done.  Clearer that way, yes.